app/views/tasks/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1> New page </h1>
+
+<%= form_with model: @task, class: 'task-form' do |f| %>
```

Comment:
Note that the html for `new` and `edit` are quite similar. This is a great place for partial views.